src/movies.js

```diff
@@ -1,16 +1,94 @@
 // Iteration 1: All directors? - Get the array of all directors.
+
+function getAllDirectors(movies) {
+    return movies.map(function (movie) {
+        return movie.director
+    })
+};
+
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
 
+function howManyMovies(array) {
+    let count = 0;
+    let dramaMovies = movies.filter(function (movie) {
+      if (movie.genre.includes("Drama")) {
+        return movie;
+      } else {
+          return 0; 
+      }
+    });
+    let spielbergDrama = dramaMovies.filter(function (movie) {
+      if (movie.director.includes("Spielberg")) {
+        count += 1;
+      }
+    });
+    return count;
+  }
```

Comment:
following the logic that you created for this interaction, this is one of the possible solution
```function howManyMovies(array) {
  let count = 0;
  let dramaMovies = array.filter(function (movie) {
    if (movie.genre.includes("Drama")) {
      return movie;
    }
  });
  let spielbergDrama = dramaMovies.filter(function (movie) {
    if (movie.director === "Steven Spielberg") {
      count += 1;
    }
  });
  return count;
}
```
instead of using the method includes() to check the string "Steven" you should use the boolean operators === to check if the value of the propriety director is the same as the string "Steven Spielberg".

Reply:
Okay thanks Jeff. I thought with includes() I would make sure that it also gets it if there are more directors. But if this is also possible with === I think it is more reliable :)